test/e2e/topology_updater/topology_updater.go

```diff
@@ -54,18 +51,15 @@ var _ = ginkgo.Describe("[TopologyUpdater][InfraConsuming] Node topology updater
 		timeout             time.Duration
 		tmPolicy            string
 		tmScope             string
-		topologyClient      *topologyclientset.Clientset
 		topologyUpdaterNode *corev1.Node
 		workerNodes         []corev1.Node
 	)
 
-	f := framework.NewDefaultFramework("topology-updater")
-	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
-	f.SkipNamespaceCreation = true
+	f := &fixture.F
```

Comment:
this is nicer than before, but shouldn't we get a new `fixture.Fixture` (thus a namespace) for each testcase, possibly in `BeforeEach`?

Reply:
The `Fixture` object is a global object, initialized once, so whether it would lie in a `BeforeEach` or not it doesn't matter.
The reason I didn't add it in the `BeforeEach` is because I didn't want to imply this object needs re initialization for every run.
The namespace creation is a separate operation that done for every test case by the fact we run it in `BeforeEach`